.github/workflows/push.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       run: sudo make pull-scripts
 
     - name: Pull in all relevant branches
-      run: git fetch origin dev-v2.5-source dev-v2.5 release-v2.5
+      run: git fetch origin dev-v2.5-source dev-v2.5 release-v2.5 dev-v2.6 release-v2.6
```

Comment:
Do we need release-v2.6 in there ?

Reply:
We can drop it. I just added it to be consistent, but technically we should drop release-v2.5 as well since its not needed